### PR TITLE
Automation Bad Day

### PR DESCRIPTION
**Pull request by wpschaub**
Transparent JOurney

### CODE REVIEW

File: content/azure-devops-automation-bad-day.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+> ![timeline](../images/azure-devops-automation-bad-day-2.png)
+
+Eventually after a barrage of tweets, Teams chats, and raising Microsoft Support tickets the [Azure DevOps Status](https://status.dev.azure.com/) switched to an ```unhealthy``` status due to the [Pipeline error: Failure in sending the provision message](https://status.dev.azure.com/_event/371776903) incident. Little did we know that this incident actually reduced the blast radius of our bad day, as most teams opted for another day by the time the Azure Agents were back to a healthy state. 
+
+---
+
+# 2AM CALL @ high noon
+
+Just before high-noon we were getting pulled into **all hands-on deck**, also known as the **infamous 2AM-call** collaboration sessions. One of our Azure Pipelines had deployed all the way to production after getting the ```thumbs-up``` from our security engineers, skipping our **Staging (ST)** and **Production (PR)** pre-approval gates. By passing change control is a severe issue. Luckily our operational support engineers delivered their usual exceptional operational support and reverted the incorrect production deployment within minutes to minimize negative end-user impact.
```
Comment (MerlinDarkstar) [inline content/azure-devops-automation-bad-day.md]: By passing  --> bypassing

**Review by MerlinDarkstar — DISMISSED** [review]

**Comment by MerlinDarkstar** [comment]
Good read, great transparency

**Comment by danielrbroderick** [comment]
Looks pretty good.

**Comment by wpschaub** [comment]
@danielrbroderick  and @MerlinDarkstar please review and make improvement suggestions. If OK, please approve. I will wait until all three reviewers have approved.

**Review by danielrbroderick — DISMISSED** [review]

**Review by MerlinDarkstar — DISMISSED** [review]

**Review by AndreKaminski — COMMENTED** [review]
This is great, but I would remove the following part "after getting the thumbs-up from our security engineers" - we don't want to throw under the bus anybody. Also, I am hoping that your screenshots contain only demo files, and no production paths.

**Comment by wpschaub** [comment]
Good point about security engineers, as it could be misinterpreted. Removed.

Screenshots are intentionally all made up with dummy data 😊

Willy

**Pull request merged**